README.md

```diff
@@ -4,3 +4,86 @@
 
 Utility to retrieve and write to the BrainIO collection of stimulus sets and assemblies.
 
+
+# BrainIO contrib
```

Comment:
rename to "Packaging Instructions"? Maybe also fix README structure, the sentence above makes it a little weird -- could have a separate "BrainIO collection" header at the very top with some description underneath